src/decorators/clamp.ts

```diff
@@ -0,0 +1,12 @@
+export function clamp(num: number, lower: number, upper: number) {
```

Comment:
Can we also use the `lodash.clamp` package here?

P.S. IMO it would be safe to just use `Math.max(lower, Math.min(upper, num))` here.

Reply:
I've updated the PR to use `lodash.clamp` instead